bluebinder.c

```diff
@@ -72,6 +72,7 @@ struct proxy {
     int gio_channel_event_id;
 
     int binder_replies_pending;
+    gboolean is_bt_failed_on_init;
```

Comment:
I suggest to call this field `init_failed`

Reply:
agreed